templates/objective-c/README.md

```diff
@@ -0,0 +1,74 @@
+# PROJECT
+
+[![CI Status](http://img.shields.io/travis/fulldecent/PROJECT.svg?style=flat)](https://travis-ci.org/fulldecent/PROJECT)
+[![Version](https://img.shields.io/cocoapods/v/PROJECT.svg?style=flat)](http://cocoadocs.org/docsets/PROJECT)
+[![License](https://img.shields.io/cocoapods/l/PROJECT.svg?style=flat)](http://cocoadocs.org/docsets/PROJECT)
+[![Platform](https://img.shields.io/cocoapods/p/PROJECT.svg?style=flat)](http://cocoadocs.org/docsets/PROJECT)
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+
+<a href="placehold.it/400?text=Screen+shot"><img width=200 height=200 src="placehold.it/400?text=Screen+shot" alt="Screenshot" /></a>
+
+
+## Example
+
+To run the example project, clone the repo, and run `pod install` from the Example directory first.
+
+
+## Requirements
+
+
+## Installation
+
+### CocoaPods
+
+[CocoaPods](http://cocoapods.org) is a dependency manager for Cocoa projects. You can install it with the following command:
+
+```bash
+$ gem install cocoapods
+```
+
+To integrate PROJECT into your Xcode project using CocoaPods, specify it in your `Podfile`:
+
+```ruby
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '9.0'
+use_frameworks!
```

Comment:
these two are optional, and are mostly not required - should probably be removed

```
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '9.0'
```